lib/template.html

```diff
@@ -126,12 +126,6 @@
 {% if site.twitter %}
   <meta name="twitter:card" content="summary" />
   <meta name="twitter:site" content="@{{ site.twitter.username | replace:"@","" }}" />
-  <meta name="twitter:title" content="{{ seo_title }}" />
-  <meta name="twitter:description" content="{{ seo_description }}" />
-
-  {% if page.image %}
-    <meta name="twitter:image" content="{{ page.image | prepend: seo_url | escape }}" />
```

Comment:
Twitter will use `og:image` instead, no need to duplicate the same info here :+1: